Sources/BraintreePaymentFlow/BTLocalPaymentRequest.swift

```diff
@@ -63,6 +63,9 @@ import BraintreeDataCollector
     var paymentID: String?
     weak var paymentFlowClientDelegate: BTPaymentFlowClientDelegate?
     var correlationID: String?
+    var paymentFlowAPIClient: BTAPIClient? {
+        return paymentFlowClientDelegate?.apiClient()
```

Comment:
We can rely on an implicit return here and remove the `return` keyword. This has been our preference throughout the SDK.

Reply:
We should style-guide that about implicit `return`

Reply:
It's done here.

Reply:
> We should style-guide that about implicit `return`

Added a note about implicit return to the [style guide](https://hackmd.io/kIyOnIFIRrKY4b5wQ0dUpw). Also added a section on use of `self` since we've talked about it here and there but not actually wrote it down.